after/ftplugin/python/jedi.vim

```diff
@@ -9,15 +9,15 @@ if g:jedi#auto_initialization
         setlocal omnifunc=jedi#completions
 
         " map ctrl+space for autocompletion
-        if g:jedi#completions_command == "<C-Space>"
+        if g:jedi#completions_command ==# '<C-Space>'
```

Comment:
Looks like javascript. Wtf. But I see why one would use it after reading the docs.